includes/export/class-wc-admin-report-csv-exporter.php

```diff
@@ -110,7 +114,11 @@ protected function map_report_controller() {
 		);
 
 		if ( isset( $controller_map[ $this->report_type ] ) ) {
-			// @todo: load the controllers if accessing outside a context where the REST API is loaded?
+			// Load the controllers if accessing outside the REST API.
+			if ( ! did_action( 'rest_api_init' ) ) {
```

Comment:
Do you know what would be calling this outside of the REST API context?

Reply:
Anything using the `WC_Admin_Report_CSV_Exporter` class directly. For example, we could build a WP-CLI command for exporting.

Reply:
Ahhh, great idea, WP CLI would be a fantastic use case.